lib/adjust/index.js

```diff
@@ -0,0 +1,9 @@
+import curryN from '../curryN'
+import toArray from '../toArray'
+
+export default curryN(3, function adgust(fn, i, arr) {
+  var a = toArray(arr)
+  return a[i]
+    ? a.slice(0, i).concat(fn(a[i]), a.slice(++i))
```

Comment:
splice here? it would be shorter

Reply:
It would mutate array. Do we need it?

Reply:
> It would mutate array. Do we need it?
Of course, not

Reply:
@BabkinAleksandr @Kelin2025  it would, but not original, as we call slice inside toArray

Reply:
Yep, I forgot about it